### ISSUE TITLE
Addition of a Squared Exponential Kernel?

### ISSUE THREAD

**Issue by mathDR** [AaltoML/BayesNewton#10]
This is not an issue per se, but I was wondering if there was a specific reason that there wasn't a Squared Exponential kernel as part of the package?

If applicable, I would be happy to submit a PR adding one.

Just let me know.

**Comment by wil-j-wil** [comment]
Hi. No good reason other than that I usually use the MarkovGP models for my research, and the SE kernel is a bit tricker to implement in that case because it doesn't have an exact form.

But I'd be very happy for you to make a PR for the standard SE kernel.

**Comment by mathDR** [comment]
Okay I will do a PR with the vanilla SE kernel (and we can always add to it later for other use cases)>. I will be sure to throw a `NotImplementedError` when needed...

**Comment by mathDR** [comment]
oh, it was denying my PR when I tried to push it.  
I got a 
```
remote: Permission to AaltoML/BayesNewton.git denied to mathDR.
```

I think you have to allow me to push PRs

**Comment by wil-j-wil** [comment]
Sorry about that - you should have access now.

**Comment by mathDR** [comment]
Okay submitted a PR [here](https://github.com/AaltoML/BayesNewton/pull/11)

**Comment by mathDR** [comment]
Added in https://github.com/AaltoML/BayesNewton/pull/11

**Comment by wil-j-wil** [comment]
Closing now since solved by #11
